content/millennium/r4/entities/location.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Location | R4 API
+---
+
+# Location
+
+* TOC
+{:toc}
+
+## Overview
+
+The Location resource describes physical places where healthcare services are provided.  In Millennium, facilities are the top level of the patient location hierarchy.  A facility is also an Organization resource at which patient locations are associated.  All facilities are organizations, but not all organizations are facilities.  A location can also be an ambulatory patient care area like a clinic or an emergency room.  The location hierarchy from highest to lowest is facility, building, nursing unit, room, and bed.
+
+
+The following fields are returned if valued:
+
+* [Location id](http://hl7.org/fhir/r4/resource-definitions.html#Resource.id){:target="_blank"}
```

Comment:
location id -> id?

Reply:
This is done as per this Jira:
https://jira2.cerner.com/browse/OPENSVC-15097